### PR TITLE
BN-16 feat: A seller should be able to delete a specific item

### PR DESCRIPTION
**Pull request by kunda4**
# PR Description
As a Seller, I should be able to delete a specific item in my colletion
Such that I can remove items I no longer deal in/were errornously added
# Description of tasks that were expected to be completed
List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style
- [x] Create delete request to delete a specific item in a given seller's collection
- [x] When the requesting user/token is a seller and the target item does not exist in the particular seller's collection and the request body is validated 
Then: 
A json object with a status: 4xx, error: the appropriate error message should be sent
- [x] When the requesting user/token is a seller and the target item already exist in the particular seller's collection and the request body is validated
Then:
The item should be deleted
A json object containing status: the appropriate code, message: appropriate response message, and item: an object of the deleted item should returned
- [x] When the request body fails any validation checks or the token is not of the role seller
Then:
The appropriate 4XX response message should be sent to the requesting client describing the validation error
# Description of tasks that were expected to be completed and that are not yet completed
- [x] none
# How this may be tested?
Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.
1. run `npm i` to install all required dependencies
2. run `npm run dev`
3. step 3
4. step 4...
# Screenshots (If appropriate/applicable)
Provide screenshots related to your PR in case it's appropriate and applicable
# Please check this Checklist before you submit your PR:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

### CODE REVIEW

**Review by Niyobuzima — DISMISSED** [review]
LGTM

**Review by uwdjany — DISMISSED** [review]
well done

**Review by dushimeemma — COMMENTED** [review]
Don't forget to write unit test @kunda4 otherwise good job!

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
 .env
 node_modules
+src/routes/vendor/userRoutesV.js
```
Comment (dushimeemma) [inline .gitignore]: why did you added this file in .gitignore?

**Review by dushimeemma — COMMENTED** [review]
If you can be able to swatch commits, this PR is ready to be merged @kunda4

**Comment by kunda4** [comment]
I will work on Swatch the commit and also removing . ignore file soon as
possible but i will not be able to attend standup for today I'm in person
case that take me more time than I expected and I don't have laptop for now.

**Pull request closed without merging**